packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1811,6 +1821,19 @@ export class Combobox
                 />
               )}
             <slot />
+            {this.noMatches === "add" && (
+              <li
+                aria-label={`${add}: ${this.filterText}`}
```

Comment:
I think this should be put into a NLS string and substituted right? Since its combining two things.

```json
"add": "Add: {text}",
```